app/models/concerns/smooch_messages.rb

```diff
@@ -430,6 +430,7 @@ def smoooch_post_save_message_actions(message, associated, app_id, author, reque
     end
 
     def smooch_relate_items_for_same_message(message, associated, app_id, author, request_type, associated_obj)
+      return unless associated_obj.is_a?(ProjectMedia)
```

Comment:
I believe it's better to move the conditions before this line(https://github.com/meedan/check-api/blob/develop/app/models/concerns/smooch_messages.rb#L459) and check associated object type